### PR TITLE
Add fields in Alarm and Situation

### PR DESCRIPTION
**Pull request by BenjaminJ**
ALEC-173

### CODE REVIEW

File: datasource/common/src/main/java/org/opennms/alec/datasource/common/ImmutableAlarm.java [hunk 1]
```diff
@@ -228,7 +274,10 @@ public String toString() {
                 ", inventoryObjectType='" + inventoryObjectType + '\'' +
                 ", summary='" + summary + '\'' +
                 ", description='" + description + '\'' +
-                ", nodeId=" + nodeId +
+                ", nodeId=" + nodeId + '\'' +
```
Comment (aramosvizcarra) [inline datasource/common/src/main/java/org/opennms/alec/datasource/common/ImmutableAlarm.java]: I think you are missing some ' after the = for nodeId, nodeLabel, nodeLocation

**Review by aramosvizcarra — CHANGES_REQUESTED** [review]
Just a minor change. Otherwise, looks good to me!

File: datasource/common/src/main/java/org/opennms/alec/datasource/common/ImmutableAlarm.java (hunk 1 above)
Comment (BenjaminJ) [reply datasource/common/src/main/java/org/opennms/alec/datasource/common/ImmutableAlarm.java]: It was missing for other fields too, I regenerated the toString with StringJoiner

**Review by aramosvizcarra — APPROVED** [review]
LGTM!

**Pull request merged**